src/app/styles.tsx

```diff
@@ -0,0 +1,139 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { openSans } from '@/styles/fonts';
+import image from '../../public/HomePage.webp';
+
+export const PageContainer = styled.div`
+  width: 100%;
+  min-height: 100vh;
+`;
+
+export const TitleSection = styled.div`
+  width: 100%;
+  height: 100vh; // Use 100vh to make the div fill the entire viewport height
+  background: linear-gradient(
+      222deg,
+      rgba(40, 120, 194, 0.5) 14.2%,
+      rgba(0, 50, 96, 0.5) 76.27%
+    ),
+    url(${image.src}) no-repeat center / cover;
+  display: flex;
+  flex-direction: column;
+  padding-top: 200px;
+  padding-right: 440px;
+  padding-left: 84px;
+  gap: 30px;
+`;
+
+export const ButtonDiv = styled.div`
+  width: 244px;
+  height: 40px;
+`;
```

Comment:
It's good practice to not fix `width` and `height` as screen sizes differ drastically. Rather, a better solution would be to use fractional units with pixel units as a boundary. For example, `width: clamp(224px, 25%, 500px)`.